### PR TITLE
Feature/page tree code style

### PR DESCRIPTION
**Pull request by githbq**
page-tree.ts: The underlined variable name is replaced with a lowercase hump style

### CODE REVIEW

File: libs/datasource/db-service/src/services/workspace/page-tree.ts [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
 
 export type ObserveCallback = () => void;
 
 export class PageTree extends ServiceBaseClass {
-    private async fetch_page_tree<TreeItem>(workspace: string) {
-        const workspace_db_block = await this.getWorkspaceDbBlock(workspace);
-        const page_tree_config =
-            workspace_db_block.getDecoration<TreeItem[]>(PAGE_TREE);
-        return page_tree_config;
+    private async fetchPageTree<TreeItem>(workspace: string) {
```
Comment (darkskygit) [inline libs/datasource/db-service/src/services/workspace/page-tree.ts:10]: private functions start with an underscore

File: libs/datasource/db-service/src/services/workspace/page-tree.ts [hunk 2]
```diff
@@ -52,109 +52,109 @@ export class PageTree extends ServiceBaseClass {
...
-        this.build_items_for_next_page(parent_page_id, new_page_id, pages);
-        await this.setPageTree<TreeItem>(target_workspace_id, [...pages]);
+        const pages = await this.getPageTree<TreeItem>(targetWorkspaceId);
+        this.buildItemsForNextPage(parentPageId, newPageId, pages);
+        await this.setPageTree<TreeItem>(targetWorkspaceId, [...pages]);
     }
-    private build_items_for_next_page(
-        parent_page_id: string,
-        new_page_id: string,
+    private buildItemsForNextPage(
```
Comment (darkskygit) [inline libs/datasource/db-service/src/services/workspace/page-tree.ts:93]: private functions start with an underscore

File: libs/datasource/db-service/src/services/workspace/page-tree.ts [hunk 3]
```diff
@@ -52,109 +52,109 @@ export class PageTree extends ServiceBaseClass {
...
+                    newPageId,
+                    childPage.children
                 );
             }
         }
     }
-    private build_items_for_prev_page(
-        parent_page_id: string,
-        new_page_id: string,
+    private buildItemsForPrevPage(
```
Comment (darkskygit) [inline libs/datasource/db-service/src/services/workspace/page-tree.ts:116]: private functions start with an underscore

File: libs/datasource/db-service/src/services/workspace/page-tree.ts [hunk 4]
```diff
@@ -52,109 +52,109 @@ export class PageTree extends ServiceBaseClass {
...
+                    newPageId,
+                    childPage.children
                 );
             }
         }
     }
-    private build_items_for_child_page(
-        parent_page_id: string,
-        new_page_id: string,
+    private buildItemsForChildPage(
```
Comment (darkskygit) [inline libs/datasource/db-service/src/services/workspace/page-tree.ts:139]: private functions start with an underscore

File: libs/datasource/db-service/src/services/workspace/page-tree.ts [hunk 5]
```diff
@@ -187,12 +187,12 @@ export class PageTree extends ServiceBaseClass {
...
     async unobserve({ workspace }: { workspace: string }) {
-        const workspace_db_block = await this.getWorkspaceDbBlock(workspace);
-        await this._unobserve(workspace, workspace_db_block.id);
+        const workspaceDbBlock = await this.getWorkspaceDbBlock(workspace);
+        await this._unobserve(workspace, workspaceDbBlock.id);
     }
 }
 
-async function update_tree_items_title<
+async function updateTreeItemsTitle<
```
Comment (darkskygit) [inline libs/datasource/db-service/src/services/workspace/page-tree.ts:195]: private functions start with an underscore

**Review by darkskygit — CHANGES_REQUESTED** [review]

File: libs/datasource/db-service/src/services/workspace/page-tree.ts (hunk 1 above)
Comment (githbq) [reply libs/datasource/db-service/src/services/workspace/page-tree.ts]: done

File: libs/datasource/db-service/src/services/workspace/page-tree.ts (hunk 2 above)
Comment (githbq) [reply libs/datasource/db-service/src/services/workspace/page-tree.ts]: done

File: libs/datasource/db-service/src/services/workspace/page-tree.ts (hunk 3 above)
Comment (githbq) [reply libs/datasource/db-service/src/services/workspace/page-tree.ts]: done

File: libs/datasource/db-service/src/services/workspace/page-tree.ts (hunk 4 above)
Comment (githbq) [reply libs/datasource/db-service/src/services/workspace/page-tree.ts]: done

File: libs/datasource/db-service/src/services/workspace/page-tree.ts (hunk 5 above)
Comment (githbq) [reply libs/datasource/db-service/src/services/workspace/page-tree.ts]: done

**Review by darkskygit — APPROVED** [review]

**Pull request merged**